enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/IverksettDto.kt

```diff
@@ -80,6 +70,13 @@ data class VurderingDto(
         val begrunnelse: String? = null
 )
 
+enum class AdressebeskyttelseGradering {
```

Comment:
Er den flyttet her for å unngå logikk i ef-sak for å sjekke om det er kode6/7 eller hva er årsaken? 

Burde denne flyttes til kontrakter.felles ? Sånn att den kan bli brukt flere steder?